src/provider/sequencer.ts

```diff
@@ -202,43 +203,52 @@ export class SequencerProvider implements ProviderInterface {
     const baseUrl = this.getFetchUrl(endpoint);
     const method = this.getFetchMethod(endpoint);
     const queryString = this.getQueryString(query);
-    const headers = this.getHeaders(method);
     const url = urljoin(baseUrl, endpoint, queryString);
 
+    return this.fetch(url, {
+      method,
+      body: request,
```

Comment:
We need a header in fetchEndpoint. (ex. auth)

Reply:
Note that this is a call for the provider's `fetch` method not the `fetch` module, the provider's `getHeaders` resolution is applied within it.

Reply:
Noted!